src/lib/components/BaseStateElement/index.js

```diff
@@ -22,9 +22,4 @@ export class BaseStateElement extends BaseElement {
     super.disconnectedCallback();
     store.unsubscribe(this.onStateChanged);
   }
-
-  /**
-   * @param {!Object<string, *>} state
-   */
-  onStateChanged(state) {}
```

Comment:
removing this seems like it would break line 12?

Reply:
Missed that, I'll add it back in.

Reply:
it might be worth adding back the state argument and the comment. In case folks are referencing this class to see what it does.